realm/realm-library/src/main/cpp/io_realm_ObjectStoreUserStore.cpp

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright 2016 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <jni.h>
+#include <jni_util/log.hpp>
+#include "io_realm_ObjectStoreUserStore.h"
+#include "sync/sync_manager.hpp"
+#include "sync/sync_user.hpp"
+#include "util.hpp"
+
+using namespace realm;
+
+const char* ERR_MULTIPLE_LOGGED_IN_USERS = "Cannot be called if more that one valid, logged-in user exists.";
+const char* ERR_NO_LOGGED_IN_USER = "No user logged-in yet.";
+const char* ERR_COULD_NOT_ALLOCATE_MEMORY = "Could not allocate memory to return all users.";
+
+static const std::shared_ptr<SyncUser>& currentUserOrThrow(JNIEnv *env) //throws
+{
+    std::vector<std::shared_ptr<SyncUser>> all_users = SyncManager::shared().all_users();
+    if (all_users.size() > 1) {
+        throw std::runtime_error(ERR_MULTIPLE_LOGGED_IN_USERS);
+    } else if (all_users.size() < 1) {
+        throw std::runtime_error(ERR_NO_LOGGED_IN_USER);
+    } else {
+        return all_users.front();
+    }
+}
+
+/*
+ * Class:     io_realm_ObjectStoreUserStore
+ * Method:    getCurrentUser
+ * Signature: ()Ljava/lang/String;
+ */
+JNIEXPORT jstring JNICALL Java_io_realm_ObjectStoreUserStore_getCurrentUser (JNIEnv *env, jclass)
```

Comment:
define the function like:
```
JNIEXPORT jstring JNICALL
Java_io_realm_ObjectStoreUserStore_getCurrentUser (JNIEnv *env, jclass) // new line for this.
```

Also, remove the comments before the function maybe? It will be generated in the header file and we don't have it for all of our current jni cpp files.

Also apply for other functions.